utils.py

```diff
@@ -3350,6 +3353,7 @@ def getDatasetFileArray( dataset, validFileOnly=0, detail=False, cache_timeout=3
 
 def getDatasetFileFraction( dataset, files):
     # VK TODO: can be replaced with dataset summary API which returns total number of evnts, files
+    # JRV: this requires to count only valid files.
```

Comment:
Jean-Roch, as far I remember DBS summary API counts only valid files. It is definitely you should consult with Yuyi which API to use here.